### PR TITLE
add player score element to read/update

### PR DESCRIPTION
**Pull request by AdamJ335-T2**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import logging
 
 import requests
+import re
```
Comment (jrsmth-tier2) [inline src/app.py:5]: For the re regex package, does it need to be added to req.txt?
(did you pip install it)

File: src/app.py [hunk 2]
```diff
@@ -156,6 +160,20 @@ def get_streak(event):
     print(streak)
     return int(streak)
 
+def get_score(event):
+    #get score value in text list
+    blocks = event.get("event").get("blocks")
+    elements = blocks[0].get("elements")
+    elements = elements[0].get("elements")
+    score_text = [elem for elem in elements if ('text' in elem and "/5" in elem.get("text"))][0]
```
Comment (jrsmth-tier2) [inline src/app.py:168]: What is `/5`?

**Review by jrsmth-tier2 — COMMENTED** [review]
Looking good, just a couple of comments

File: src/app.py (hunk 1 above)
Comment (AdamJ335-T2) [reply src/app.py:5]: The re package incorporated to Python as standard. No installations needed:
https://www.freecodecamp.org/news/how-to-import-a-regular-expression-in-python/#:~:text=It%20is%20a%20standard%20module,it%20separately%20with%20package%20managers.

File: src/app.py (hunk 2 above)
Comment (AdamJ335-T2) [reply src/app.py:168]: We are grabbing the element that contains the score. In the format of the wafflegame
```
#waffle727 4/5
:large_green_square::large_green_square::large_green_square::large_green_square::large_green_square:
:large_green_square::star::large_green_square::star::large_green_square:
:large_green_square::large_green_square::large_green_square::large_green_square::large_green_square:
:large_green_square::star::large_green_square::star::large_green_square:
:large_green_square::large_green
```
The element in question will be 
#waffle727 4/5
So the score will always been in the element text block with "/5"

**Pull request merged**